### ISSUE TITLE
it just keeps scrolling

### ISSUE THREAD

**Issue by tboss12310** [MetaChar/Hatch#7]
I have everything setup and good but when I run it I get to the correct password and nothing it just keeps scrolling by

**Comment by tarikbakk** [comment]
this is what happened to me as well,any solution or it just doesn't work...

**Comment by 0vertime-dev** [comment]
Yes, all I did was add an extra selector for the error message after you get a wrong password worked perfectly

**Comment by tarikbakk** [comment]
Thanks for your prompt response, but in my case I don't get any error message when I try a wrong password, anything else I can do?? Just so you know I'm brute-forcing a router login page 😊

**Comment by 0vertime-dev** [comment]
Do you have a picture of what the page looks like/ what router page are you
brute forcing?

**Comment by tarikbakk** [comment]
![Screenshot_20200719_234808](https://user-images.githubusercontent.com/68480555/87888150-79d23080-ca1a-11ea-8ca8-2ec209750d42.jpg)

**Comment by tarikbakk** [comment]
It's a technicolor router
